service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/docmosis/AllOrdersDocmosisTest.java

```diff
@@ -0,0 +1,143 @@
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.docmosis.generator.DocmosisOrderCaseDataGenerator;
+import uk.gov.hmcts.reform.fpl.enums.OrderStatus;
+import uk.gov.hmcts.reform.fpl.enums.docmosis.RenderFormat;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.DocmosisDocument;
+import uk.gov.hmcts.reform.fpl.model.order.Order;
+import uk.gov.hmcts.reform.fpl.service.CaseDataExtractionService;
+import uk.gov.hmcts.reform.fpl.service.ChildrenService;
+import uk.gov.hmcts.reform.fpl.service.DocumentDownloadService;
+import uk.gov.hmcts.reform.fpl.service.HearingVenueLookUpService;
+import uk.gov.hmcts.reform.fpl.service.UploadDocumentService;
+import uk.gov.hmcts.reform.fpl.service.docmosis.DocmosisDocumentGeneratorService;
+import uk.gov.hmcts.reform.fpl.service.docmosis.DocumentConversionService;
+import uk.gov.hmcts.reform.fpl.service.orders.OrderCreationService;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.C21BlankOrderDocumentParameterGenerator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.C23EPOAdditionalDocumentsCollector;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.C23EPODocumentParameterGenerator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.C32CareOrderDocumentParameterGenerator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.C33InterimCareOrderDocumentParameterGenerator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.C35aSupervisionOrderDocumentParameterGenerator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.C47AAppointmentOfAChildrensGuardianParameterGenerator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.DocmosisCommonElementDecorator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.DocumentMerger;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.OrderDocumentGenerator;
+import uk.gov.hmcts.reform.fpl.service.orders.generator.OrderDocumentGeneratorHolder;
+import uk.gov.hmcts.reform.fpl.utils.captor.ResultsCaptor;
+
+import java.util.Set;
+import java.util.stream.Stream;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.docmosis.DocmosisHelper.extractPdfContent;
+import static uk.gov.hmcts.reform.fpl.docmosis.DocmosisHelper.remove;
+import static uk.gov.hmcts.reform.fpl.utils.ResourceReader.readString;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocument;
+
+@SpringBootTest(classes = {
```

Comment:
I would move `ObjectMapper.class`, `DocmosisDocumentGeneratorService.class`, `RestTemplate.class` and any other common classes to the base class like we do with the notification tests, then to access them in this class you just need to use `@ContextConfiguration` instead of `@SpringBootTest`